components/tidb_query/src/expr/builtin_string.rs

```diff
@@ -492,17 +483,13 @@ impl ScalarFunc {
         if self.field_type.get_flen() == -1
             || self.field_type.get_flen() > tidb_query_datatype::MAX_BLOB_WIDTH
         {
-            return Ok(Some(Cow::Borrowed(b"")));
+            return Ok(Some(Cow::Owned(b"".to_vec())));
```

Comment:
Please help me change this to return `Ok(None)`. It's a bug I introduced before. thanks

Reply:
sorry, i forgot it🤦‍♀️